sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -373,7 +374,43 @@ public void waitUntilNumberOfElementsIsReduceToGiven(By selector, int given) {
 
   public String getCheckedOptionFromHorizontalOptionGroup(By options) {
     waitUntilIdentifiedElementIsPresent(options);
-    return baseSteps.getDriver().findElement(options).findElement(SELECTED_RADIO_BUTTON).getText();
+    if (baseSteps.getDriver().findElement(options).findElements(SELECTED_RADIO_BUTTON).size()
+        == 0) {
+      return null;
+    } else {
+      return baseSteps
+          .getDriver()
+          .findElement(options)
+          .findElement(SELECTED_RADIO_BUTTON)
+          .getText();
+    }
```

Comment:
Please do not change the method if you are using it for vertical options

Reply:
@BojteTamas, the getCheckedOptionFromHorizontalOptionGroup crashes with NPE if there isn't any element selected. this is why i was adding that catch.

in the future we shall extend it further for getting multiple selection. currently it will return only the first selection. 

please let me know if you still want to change it back. I will have to create a new method for getting edited values. i was extending and using the one created by @rdutu-vg. it will create some duplicate code if yes.

Reply:
Avoid creating new methods to get the size of an elements list and get text, we already have: **getNumberOfElements()**  and **getTextFromPresentWebElement()**

Reply:
Better create a new method please

Reply:
@BojteTamas created getOptionFromHorizontalGroupOrNullIfNothingChecked
@rdutu-vg those method are not applicable here. I'm using two elements for getting the selection.

Reply:
rezolved imho